streams/src/test/java/org/apache/kafka/streams/state/internals/SessionBytesStoreTest.java

```diff
@@ -472,6 +472,11 @@ public void shouldLogAndMeasureExpiredRecords() {
         assertThat(messages, hasItem("Skipping record for expired segment."));
     }
 
+    @Test
+    public void shouldNotThrowRemovingNonexistentKey() {
+        sessionStore.put(new Windowed<>("a", new SessionWindow(0, 1)), 0L);
```

Comment:
Should this be `sessionStore.remove` (`remove()` is called within `put()` atm, but it seem to be better to call `remove()` directly in the unit test)?

Reply:
Yeah that should be `remove`